index.bs

```diff
@@ -34,23 +34,23 @@ spec: RFC6455; urlPrefix: https://tools.ietf.org/html/rfc6455
     text: Fail the WebSocket Connection; url: section-7.1.7
     text: Status Codes; url: section-7.4
     text: Handling Errors in UTF-8-Encoded Data; url: section-8.1
-spec: RFC8610; urlPrefix: https://tools.ietf.org/html/rfc8610
+spec: RFC8610; urlPrefix: https://datatracker.ietf.org/doc/html/rfc6455#section-3
```

Comment:
Having the fragment in the URL here seems unusual. Is it intentional?

Reply:
No, and even more importantly, that link targets RFC6455 whereas it should go to RFC8610. I'll fix that!

Reply:
I was very much missing the forest for the trees in the review then, but thanks for fixing correctly anyway :)